Packs/FeedCrowdstrikeFalconIntel/Integrations/FeedCrowdstrikeFalconIntel/FeedCrowdstrikeFalconIntel.py

```diff
@@ -0,0 +1,273 @@
+
+        return parsed_indicators
+
+    def build_actors_filter(self, target_countries, target_industries, custom_filter):
+        actors_filter = ''
+        if custom_filter:
+            actors_filter = custom_filter
+            return actors_filter
+
+        if target_countries:
+            target_countries = target_countries.split(',')
+            target_countries = [country.replace(' ', '%20') for country in target_countries if len(country) > 1]
+            for country in target_countries:
+                actors_filter += f'target_countries%3A"{country}"%2B'
+
+        if target_industries:
+            target_industries = target_industries.split(',')
+            target_industries = [industry.replace(' ', '%20') for industry in target_industries if len(industry) > 1]
+            for industry in target_industries:
+                actors_filter += f'target_industries%3A"{industry}"%2B'
+
+        if actors_filter:
+            actors_filter = '?filter=' + actors_filter[:-3]
+        return actors_filter
+
+    def build_url_suffix(self, params, actors_filter):
+        url_suffix = "/intel/combined/actors/v1"
+        if actors_filter:
+            url_suffix = url_suffix + actors_filter
+            if params:
+                url_suffix = url_suffix + '%2B' + params
+        elif params:
+            url_suffix = url_suffix + '?filter=' + params
+        return url_suffix
+
+    def build_iterator(self, feed_tags: List, limit=None, offset=None, target_countries=None, target_industries=None,
+                       custom_filter=None):
+        """Builds a list of indicators.
+        Returns:
+            list. A list of JSON objects representing indicators fetched from a feed.
+        """
+        actors_filter = self.build_actors_filter(target_countries, target_industries, custom_filter)
+
+        # for fetch_indicator_command only
+        params = {}
+        if not limit:
+            params = self.get_last_modified_time()
+
+        suffix_url_to_filter_by = self.build_url_suffix(params, actors_filter)
+        response = self.check_quota_status(suffix_url_to_filter_by)
+        parsed_indicators = self.create_indicators_from_response(response, feed_tags)  # list of dict of indicators
+
+        # for get_indicator_command only
+        if limit:
+            parsed_indicators = parsed_indicators[int(offset): int(offset) + int(limit)]
+        return parsed_indicators
+
+    def set_last_modified_time(self):
+        current_time = datetime.now()
+        current_timestamp = datetime.timestamp(current_time)
+        timestamp = str(int(current_timestamp))
+        integration_context_to_set = {'last_modified_time': timestamp}
+        demisto.setIntegrationContext(integration_context_to_set)
+
+    def get_last_modified_time(self):
+        demisto.setIntegrationContext(None)
+        integration_context = demisto.getIntegrationContext()
+        if not integration_context:
```

Comment:
Because this api does not return much information, in the first time we return everything that is in the api and in the next times we return only what has changed there